R/simulation-batch.R

```diff
@@ -71,6 +71,25 @@ SimulationBatch <- R6::R6Class(
       batchRunValues <- SimulationBatchRunValues$new(parameterValues, initialValues)
       rClr::clrCall(self$ref, "AddSimulationBatchRunValues", batchRunValues$ref)
       return(batchRunValues$id)
+    },
+
+    #' @description
+    #' Print the object to the console
+    #' @param ... Rest arguments.
```

Comment:
```suggestion
    #' @param ... Rest of the arguments.
```

Reply:
isn't it called in fact Rest argument or (Rest parameter)? Or is it just in Javascript that it is called like this? Just curious

Reply:
Not in R, no. 

Typically, it is documented as 

- for `S3` methods (e.g., see `?base::mean`, `?tidyr::extract`, etc.)

```r
... further arguments passed to or from other methods
```

- for regular functions

```r
... Additional arguments
```

No hard and fast rules here, of course. 

So, if `Rest arguments` is the convention you have adopted, ignore my suggestion, but it's definitely not idiomatic of R package docs.